doc/newsfragments/2637_changed.fix_get_between.rst

```diff
@@ -0,0 +1 @@
+:py:meth:`~testplan.common.utils.match.LogMatcher.get_between` would raise an exception upon binary configuration of the matcher.
```

Comment:
Is not this change supposed to fix this?

Reply:
Ouch, double click erased some stuff. Thanks for catching.